### PR TITLE
[BN] Translation of week 12

### PR DESCRIPTION
**Pull request by bhuiyanmobasshir94**
Added the Bengali translation for week 12 and part 0. Also updated the config file path.

### CODE REVIEW

**Review by Atcold — APPROVED** [review]
Great, thanks!

File: docs/bn/week12/12.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+lang-ref: ch.12
+title: Week 12
```
Comment (Atcold) [inline docs/bn/week12/12.md:3]: You forgot to translate the title.

**Review by Atcold — CHANGES_REQUESTED** [review]
Please, translate the title as well.

**Review by kishordgupta — CHANGES_REQUESTED** [review]
সূচনা করব - is getting repeated,
How about use some synonom
Like  এরপর শুরু করব / তারপরে পরিচিত হব etc

File: docs/bn/week12/12.md (hunk 1 above)
Comment (bhuiyanmobasshir94) [reply docs/bn/week12/12.md:3]: Ok sure

**Review by kishordgupta — APPROVED** [review]

**Comment by bhuiyanmobasshir94** [comment]
I think conflicts have been resolved here for this PR. Thank you.

**Review by Atcold — APPROVED** [review]
Great, thanks!

**Pull request merged**